external/webserver/http_client.c

```diff
@@ -871,6 +871,187 @@ void http_handle_file(struct http_client_t *client, int method, const char *url,
 	}
 }
 
+static int prepare_chunk_body(char *buf, int len, const char *body, bool is_last_msg)
+{
+	int chunk_size;
+	char httpcrnl[] = "\r\n";
+	int buflen = 0;
+
+	buflen = len;
+
+	if (is_last_msg) {
+		chunk_size = 0;
+	} else {
+		if (body == NULL) {
+			HTTP_LOGE("msg body is NULL \n");
+			return -1;
+		}
+		chunk_size = strlen(body);
```

Comment:
Please include `string.h`.

Reply:
It is recommended to include each header into each file which the function used.

Reply:
Is this coding guideline mentioned anywhere in TizenRT ?